apstra/api_property_sets_test.go

```diff
@@ -55,75 +55,60 @@ func TestCreateGetUpdateGetDeletePropertySet(t *testing.T) {
 	}
 
 	for clientName, client := range clients {
-		log.Printf("testing createPropertySet() against %s %s (%s)", client.clientType, clientName, client.client.ApiVersion())
-		id1, err := client.client.createPropertySet(ctx, ps)
+		log.Printf("testing CreatePropertySet() against %s %s (%s)", client.clientType, clientName, client.client.ApiVersion())
+		id1, err := client.client.CreatePropertySet(ctx, ps)
 		if err != nil {
 			t.Fatal(err)
 		}
-
-		log.Printf("testing getPropertySetsByLabel() against %s %s (%s)", client.clientType, clientName, client.client.ApiVersion())
-		psSlice, err := client.client.GetPropertySetsByLabel(ctx, ps.Label)
+		log.Printf("testing GetPropertySetByLabel() against %s %s (%s)", client.clientType, clientName, client.client.ApiVersion())
+		ps1, err := client.client.GetPropertySetByLabel(ctx, ps.Label)
 		if err != nil {
-			t.Fatal(err)
-		}
-		if len(psSlice) != 1 {
-			t.Fatalf("expected 1 property set, got %d", len(psSlice))
+			log.Printf("Error Received %q", err)
 		}
 
-		log.Printf("testing getPropertySetByLabel() against %s %s (%s)", client.clientType, clientName, client.client.ApiVersion())
-		_, err = client.client.getPropertySetByLabel(ctx, ps.Label)
-		if err != nil {
-			t.Fatal(err)
+		if !equal((*PropertySetData)(ps), ps1.Data) {
+			t.Fatal("Created and extracted objects are not equal")
 		}
 
-		log.Printf("testing getPropertySet() against %s %s (%s)", client.clientType, clientName, client.client.ApiVersion())
-		ps1, err := client.client.getPropertySet(ctx, id1)
-		if err != nil {
-			t.Fatal(err)
-		}
-		polished1, err := ps1.polish()
-		if err != nil {
-			t.Fatal(err)
-		}
-		if !equal(ps, polished1.Data) {
-			t.Fatal("ps and ps1 are not equal")
+		log.Printf("Testing duplicate CreatePropertySet() against %s %s (%s)", client.clientType, clientName, client.client.ApiVersion())
```

Comment:
Turns out the duplicate name prohibition is a 4.1.2-only thing. We need to either detect the version and skip this test for earlier apstra, or delete the test altogether, same as we have done for `GetThingsByLabel()`.

I'm leaning toward axe the test.